nucypher/blockchain/eth/constants.py

```diff
@@ -7,19 +7,15 @@ class TokenConfigError(ValueError):
         pass
 
     __subdigits = 18
-    _M = 10 ** __subdigits                                 # Unit designation
-    __initial_supply = int(1e9) * _M                       # Initial token supply
-    __saturation = int(3.89e9) * _M                        # Token supply cap
+    M = 10 ** __subdigits                                 # Unit designation
+    __initial_supply = int(1e9) * M                       # Initial token supply
+    __saturation = int(3.89e9) * M                        # Token supply cap
```

Comment:
Eh.. do these parameters also have to be with underscores?

Reply:
No; however, they are only used internally for constant calculations.

Reply:
Ah, makes sense then.
How do we _set_ those values though? I can expect them to be different on testnet vs mainnet

Reply:
Seems like these values won't ever be set during runtime..right? 
I've always felt that this module is at least somewhat configuration (in fact this class had the name `NuCypherMinerConfig` for some time).

Perhaps default values can be set and optionally used in the constant classes's `__init__` like we do with most injectable configs.  

Maybe we make subclasses if there are a few unchanging sets of constants we will use.

Reply:
This is categorically different then every other config class because Alice, Ursula, and Bob don't set/change the values for typical (and most atypical) scenarios I can imagine.  Once a network is deployed with one set of values, those values are then *required* to be compatible with the rest of the network.  Still, if we intend to change these values programmatically, its best to follow our existing dependency injection scheme.

Reply:
I am just saying that these values will be different on mainnet and testnet, possibly. Maybe it can be hardcoded for two of them, but what you select in configuration is either one preset or the other

Reply:
Got it.  Yeah - I think both subclassing *and* using default values will work well and remain extensible. :-)

Reply:
Re: the variable names, perhaps add some comments explaining why they will never be changed during runtime?